src/Emanote/Model/Stork/Index.hs

```diff
@@ -79,11 +79,16 @@ runStork input = do
       -- title (but why would they?)
       T.replace "\\\\U" "\\U"
 
-newtype Input = Input
-  { inputFiles :: [File]
+data Input = Input
+  { inputFiles :: [File],
+    inputFrontmatterHandling :: Text
```

Comment:
Instead of `Text`, I'd create a new type here:

```haskell
data Handling 
  = Handling_Ignore
  | Handling_Omit
  | Handling_Parse
```

... to correspond to the 3 values in https://stork-search.net/docs/config-ref#frontmatter_handling

And then use it as:

```haskell
  inputFrontmatterHandling :: Handling 
```

Reply:
Reason: https://lexi-lambda.github.io/blog/2019/11/05/parse-don-t-validate/

Specifically,

> - **Let your datatypes inform your code, don’t let your code control your datatypes.** Avoid the temptation to just stick a `Bool` in a record somewhere because it’s needed by the function you’re currently writing. Don’t be afraid to refactor code to use the right data representation—the type system will ensure you’ve covered all the places that need changing, and it will likely save you a headache later.

Reply:
Makes sense.